misc-commands/misc.js

```diff
@@ -285,6 +285,95 @@ const stopListening = (receivedMessage, timeout = 0) => {
 	}
 }
 
+const getXkcdComicInfo = async (num) => {
+	const response = await axios.get(`https://xkcd.com/${num}/info.0.json`);
+		if (response.status !== 200) {
+			// send error
+			base.sendError(receivedMessage, `Failed to get comic #${num}\n${response.status}: ${response.statusText}`)
+			return;
+		}
+		return response.data;
+}
+
+const xkcd = async (receivedMessage, args) => {
+	const requestedComic = (args[0] || "").trim();
+	if (!requestedComic) {
```

Comment:
This if statement is useless, there's a duplicate if statement 2 lines down that is the same condition?

Reply:
Good point. Fixed.